phoenix-core-client/src/main/java/org/apache/phoenix/compile/FromCompiler.java

```diff
@@ -180,7 +180,8 @@ public static ColumnResolver getResolverForCreation(final CreateTableStatement s
         NamedTableNode tableNode = NamedTableNode.create(null, baseTable, Collections.<ColumnDef>emptyList());
         // Always use non-tenant-specific connection here
         try {
-            SingleTableColumnResolver visitor = new SingleTableColumnResolver(connection, tableNode, true);
+            SingleTableColumnResolver visitor
```

Comment:
Why are we always hitting the server when resolving the table ?

Reply:
@tkhurana This creates a resolver for table creation. As discussed in PHOENIX-7271, we will always resolve before DDL operations.